egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -116,6 +116,16 @@ lang=noinfo      # The language type of corpus.
 text_fold_length=150   # fold_length for text data.
 speech_fold_length=800 # fold_length for speech data.
 
+# VERSA eval related
+skip_scoring=true # Skip scoring stages.
+skip_wer=true # Skip WER evaluation.
```

Comment:
VERSA can also do WER, we can consider using versa only, which makes the maintain effort easier?